### DISCUSSION TITLE
Numbers as model keys (need help with a schema)

### DISCUSSION THREAD

**Discussion by asobirov** [prisma/prisma#13109] [Q&A] [answered]
Hey, 

I was wondering if it is possible use numbers as keys. I'm trying to create user's working hours schedule, where each day is represented as numbers (0-7), however I'm not sure if it is supported

The model could have look like this: 

```
model Schedule {
  id String @id @default(auto()) @map("_id") @db.ObjectId

  "0" WorkingHours[]
  "1" WorkingHours[]
  "2" WorkingHours[]
  "3" WorkingHours[]
  "4" WorkingHours[]
  "5" WorkingHours[]
  "6" WorkingHours[]
}

type WorkingHours {
  start String
  end   String
}
```

If you have any recommendation or idea feel free to let me know.

**Comment by asobirov** [comment]
Not sure if this is the best way to structure the model, however i used `@map` to make the keys as numbers

```model Schedule {
  id String @id @default(auto()) @map("_id") @db.ObjectId

  user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
  userId String @db.ObjectId

  sun WorkingHours[] @map("0")
  mon WorkingHours[] @map("1")
  tue WorkingHours[] @map("2")
  wed WorkingHours[] @map("3")
  thu WorkingHours[] @map("4")
  fri WorkingHours[] @map("5")
  sat WorkingHours[] @map("6")

  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

type WorkingHours {
  start String?
  end   String?
}```

**Comment by pantharshit00** [comment, marked as answer]
Hello @asobirov , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I can see you have figured out that you will need to use `@map` here. Schema grammar doesn't allow identifier to start with a number so you will need to use a map here.
